example/passthrough_hp.cc

```diff
@@ -1211,6 +1214,13 @@ static cxxopts::ParseResult parse_options(int argc, char **argv) {
     if (fs.debug || fs.debug_fuse)
         fs.foreground = true;
 
+    auto mnt_opts = options["o"].as<std::string>();
```

Comment:
I guess there should be added a comment here that this is specially for xfstests.